src/OpenOfficeConvert/OOStyles.cs

```diff
@@ -2310,7 +2307,7 @@ private void SetPageNumberInFrame()
             _writer.WriteAttributeString("fo:min-width", "35pt");
             _writer.WriteAttributeString("draw:z-index", "1");
             _writer.WriteStartElement("draw:text-box");
-            _writer.WriteAttributeString("fo:min-height", "14.14pt");//added for TD-2579
+	        _writer.WriteAttributeString("fo:min-height", "19.00pt");

```

Comment:
Here again we might need some comment to explain why the value of 19 was chosen as a fixed value. Should we get a number from a style sheet?